### DISCUSSION TITLE
Allow iframe to docs.rs

### DISCUSSION THREAD

**Discussion by SoniEx2** [rust-lang/crates.io#6093] [Feature Requests]
iframe to docs.rs is a better experience than including README.md as crate docs.

supports code links too. e.g.

```rust
//! This is a crate comment with a link to [`String`]. This doesn't work in a README.md
```

**Comment by Turbo87** [comment]
you can achieve the same thing with a regular markdown link to the Rust docs. This would be a pretty drastic change, which seems unlikely to happen at this point.
